SwanGallery/src/__tests__/SwanGallery.spec.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Example of [Jest](https://jestjs.io/docs/getting-started) unit tests
+ */
+
+describe('SwanGallery', () => {
```

Comment:
Can we remove this file if it is not used? (If you have time, you can add some tests here otherwise.)

Reply:
The test files were automatically generated, I will remove it.